### PR TITLE
Cloud January Release Notes

### PR DESCRIPTION
**Pull request by gemma-shay**
release notes january 2023

### CODE REVIEW

File: releases/cloud.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ Get future release notes emailed to you:
...
+## January 9, 2023
+
+<h3> Console changes </h3>
+
+- {{ site.data.products.dedicated }} clusters now have access to the [**Metrics** page](../v22.2/ui-custom-chart-debug-page.html#available-metrics).
+- {{ site.data.products.dedicated }} clusters running CockroachDB [v22.1.8](../releases/v22.1.html#v22-1-8) or later now have a separate tab for incomplete backup jobs on the [**Backups** page](../cockroachcloud/backups-page.html#backups-page).
+
+<h3> Cloud API changes </h3>
+
+- The [create cluster](../cockroachcloud/cloud-api.html#create-a-new-cluster) request now exposes the `RestrictEgressTraffic` boolean field to allow clusters to be created with a [deny-by-default egress traffic policy](../cockroachcloud/egress-perimeter-controls.html#use-a-deny-by-default-egress-traffic-policy).
```
Comment (abhinavg6) [inline releases/cloud.md:28]:
```suggestion
- The [create cluster](../cockroachcloud/cloud-api.html#create-a-new-cluster) request now exposes the `restrict-egress-traffic` boolean field to allow clusters to be created with a [deny-by-default egress traffic policy](../cockroachcloud/egress-perimeter-controls.html#use-a-deny-by-default-egress-traffic-policy).
```

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Requested a change to cloud API create cluster change. Thanks.

File: releases/cloud.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ Get future release notes emailed to you:
 
 {% include releases/current-cloud-version.md %}
 
+## January 9, 2023
+
+<h3> Console changes </h3>
+
+- {{ site.data.products.dedicated }} clusters now have access to the [**Metrics** page](../v22.2/ui-custom-chart-debug-page.html#available-metrics).
```
Comment (thtruo) [inline releases/cloud.md:23]: @gemma-shay we're still gating this behind a feature flag and have not opened it up for all users yet. I recommend we cut this for the Jan release note.

**Review by abhinavg6 — APPROVED** [review]

**Review by rmloveland — APPROVED** [review]
LGTM!

**Pull request merged**